setup.cfg

```diff
@@ -19,15 +19,21 @@ python_requires = >=3.6
 packages = find:
 setup_requires =
     setuptools_scm
+install_requires =
```

Comment:
As we use NumPy directly, should it be listed as a dependency?

Reply:
Good point, have added `numpy`